3001-4000/LOC3170Gadama.xml

```diff
@@ -54,7 +54,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <placeName xml:lang="gez" corresp="#n1" type="normalized">Gadāma Barāqit</placeName> 
                <placeName xml:lang="gez" xml:id="n2" type="alt">ገዳመ፡ ቃሕን፡</placeName>
                <placeName xml:lang="gez" corresp="#n2" type="normalized">Gadāma Qāḥen</placeName>
-                  
+               <placeName xml:lang="gez" xml:id="n3" type="alt">ገዳመ፡ በራቒት፡</placeName>
```

Comment:
Sorry, to get it right, where is this place? Is it the church of Beraqit with the Acts of Yem'ata?

Reply:
it is, according to the EAE "Gadama Baraqit in Qwabayn (Gadama Qaben)," mentioned in the article on Ewostatewos

Reply:
I can merge it now, if necessary, but I want to attract the attention to the possibility of having a confusion. On one hand, there is a place called "Qaḥen Däbrä Ṣadəqan" (see Ethio-Spare), an Ewostatean one (see a descrepancy above betwee fidel and transcription), I do not know if it is also called በራቒት፡ or not; on the other hand two places called  በራቒት፡ seem to exist, one in Eritrea and one with the Acts of Yem'ata in Ethiopia, Tigray (I do not know if the name Qaḥen or smth like this is attached to them).

Reply:
I don't know anything about it, I added this name to the place record created from the EAE, I have no opinion on whether it should be merged

Reply:
Then let it be open, for a while.